src/analysis/locate.ml

```diff
@@ -330,8 +328,6 @@ let uid_of_path ~config ~env ~ml_or_mli ~decl_uid path ns =
         match Misc.find_in_path_uncap (Mconfig.cmt_path config) fn with
         | filename ->
           let cmt_infos = (Cmt_cache.read filename).cmt_infos in
-          move_to filename cmt_infos;
```

Comment:
> 



> Do I understand correctly that the fix is to look at `Mconfig.cmt_path` again, instead of just `build_path` (which is what `Load_path` does)?

Yes.

Reply:
> 



> Do I understand correctly that the fix is to look at `Mconfig.cmt_path` again, instead of just `build_path` (which is what `Load_path` does)?

Yes.